spec/acceptance/basic_spec.rb

```diff
@@ -90,7 +90,7 @@ class { 'pulpcore':
   end
 
   describe command("DJANGO_SETTINGS_MODULE=pulpcore.app.settings PULP_SETTINGS=/etc/pulp/settings.py rq info -c pulpcore.rqconfig") do
-    its(:stdout) { is_expected.not_to match(/Connection refused/) }
+    its(:stdout) { is_expected.to match(/Connection refused/) }
```

Comment:
If this passed previously, I would expect this to now fail. Be interesting to see waht the results are.

Reply:
I think lack of cleanup / test ordering is impacting the results on GH.

When I run it on EL8 Vagrant image locally:

```
Finished in 5 minutes 25 seconds (files took 1 minute 34.24 seconds to load)
62 examples, 0 failures
```

My theory is that on GH, Redis is left installed and the service is running, but the connection isn't configured in settings.py

Reply:
You should indeed ensure Redis is absent before starting the test. The order is not really defined that well and you can't assume it. Another example is where you rerun tests manually.